app/templates/components/speaker-item.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="speaker-item">
+  {{#if speaker.image}}
+    <img alt="" class="ui image" src="{{speaker.image}}">
+  {{else}}
+    <img alt="" class="ui image" src="/images/placeholders/avatar.png">
+  {{/if}}
+  <br>
+  <div class="speaker-name">{{ speaker.name }}</div>
+  <div class="speaker-organisation">{{ speaker.organisation }}</div>
+  <div class="social-links">
+    {{#each speaker.socialLinks as |link|}}
+      <a href="{{link.url}}"><i class="bordered {{link.name}} colored inverted icon"></i></a>
```

Comment:
Since it's an external link, add `target="_blank" rel="noopener noreferrer"` attributes to the anchor tag.